### DISCUSSION TITLE
Ability to replicate deformable objects

### DISCUSSION THREAD

**Discussion by cold-young** [NVIDIA-Omniverse/PhysX#93] [Q&A]
Hi.
I am doing a deformable object RL in isaacsim.
The biggest feature of isaacsim simulation is that it can replicate objects and simulate them in parallel.
I want to replicate the deformable object and simulate it in parallel. 
The omniverse isaacgym has shown that the deformable object does not support replication. I wonder if this is a problem with my code or if it is completely impossible in the simulation.

If it is impossible to replicate an object, I wonder if there is an alternative way to simulate a deformable object in parallel.

Thanks,

**Comment by preist-nvidia** [comment]
Hi, could you please redirect your question to the IsaacSim forum?

https://forums.developer.nvidia.com/c/omniverse/simulation/69
